.appveyor.yml

```diff
@@ -24,7 +24,7 @@ platform:
 install:
     # If there is a newer build queued for the same PR, cancel this one.
     - cmd: |
-        powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.githubusercontent.com/conda-forge/conda-forge-ci-setup-feedstock/branch2.0/recipe/conda_forge_ci_setup/ff_ci_pr_build.py', 'ff_ci_pr_build.py')"
+        powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.githubusercontent.com/conda-forge/conda-forge-ci-setup-feedstock/master/recipe/conda_forge_ci_setup/ff_ci_pr_build.py', 'ff_ci_pr_build.py')"
```

Comment:
Those scripts SHOULD NOT be modified manually but updated using `conda smithy rerender`.

Reply:
OK, sorry. I'll try.

Reply:
The rerendering worked for me. I pushed the changes to master as directed. Assuming the master build passes: should I then rebase this PR on top of master?

Reply:
The builds passed. I rebased and force pushed. I hope this was correct. 🤞

Reply:
Yes, they recently switched everything to Azure (faster, everything at the same place and longer build are allowed).

The good practice is too never push to master directly but open a PR on **your fork** and then merge the PR.

You couldn't know this time ofc so all fine.